theme/documentation/article_pages.md

```diff
@@ -19,7 +19,7 @@ This article is to be used as a general purpose article; designed for informatio
 
 ### Knowledge base article
 
-This article is to be used to promote knowledge sharing and to provide solutions for customer issues before reaching out to customer support engineer for additional help. It contains a sidebar of **Related Articles** to increase article visibility.
+This article is to be used to promote knowledge sharing and to provide solutions for customer issues before reaching out to customer support engineer for additional help. It is very similar to the General Article template with the addition of content author underneath the heading. 
```

Comment:
Wordsmith and remove space after the period.

`It is very similar to the General Article template with the addition of the content author underneath the heading.`

Reply:
353557d